hooks/pdlc-outer-loop.sh

```
@@ -308,6 +316,23 @@ while [[ "${SESSION_COUNT}" -lt "${MAX_SESSIONS}" ]]; do
     fi
   fi
 
+  # --- Save session checkpoint (REQ-SP-003) ---
+  # Determine actor result from git diff (changes vs no changes)
+  local actor_result="no changes"
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)